src/main/java/ChatReques5.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques5 {
+    private String model;
```

Comment:
Wildcard type List<?> can lead to type safety issues. It is recommended to use a specific type instead of a wildcard.